pkg/util/errors.go

```diff
@@ -0,0 +1,10 @@
+package util
```

Comment:
i am not sure if it makes sense to have dedicated utils package just to hold error definitions. They could easily have stayed in pkg/apis/common
we already use that to hold conditions

Reply:
> i am not sure if it makes sense to have dedicated utils package just to hold error definitions. They could easily have stayed in pkg/apis/common

After further consideration, I believe it makes sense to maintain the errors in “util” because they are not related to the API itself, but are used in several places within the project. The errors are only used within the codebase internally and are not relevant for the API.